boards/shields/st_b_cams_omv_mb1683/boards/stm32h7b3i_dk.overlay

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#include <zephyr/dt-bindings/video/video-interfaces.h>
+
+&i2c4 {
+	pinctrl-0 = <&i2c4_scl_pd12 &i2c4_sda_pd13>;
+	pinctrl-names = "default";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	ov5640: ov5640@3c {
```

Comment:
This doesn't belong to the board overlay but to the shield overlay, no? 🤔
Looks like the 30-pin connector will need to be defined so that it can be used as an abstraction in the shield overlay

Reply:
Thanks! Please let me know if additional changes are required.